docker/github/__init__.py

```diff
@@ -267,4 +267,11 @@ async def deploy(
 
     @property
     async def repositories(self):
-        return await self.get("/user/repos")
+        result = []
+        page = 1
```

Comment:
Ich fände es schöner, wenn wir nicht mit einer Index-Variable loopen und den Endpunkt ansprechen, bis keine Ergebnisse mehr kommen.
Stattdessen bevorzuge ich, den Link-Header der Response auszuwerten, um die URL der nächsten Seite mit Repositories zu erhalten. Allerdings muss man dafür den Link Header in der Form 


    <https://api.github.com/user/repos?per_page=2&page=1>; rel="prev", <https://api.github.com/user/repos?per_page=2&page=3>; rel="next", <https://api.github.com/user/repos?per_page=2&page=7>; rel="last", <https://api.github.com/user/repos?per_page=2&page=1>; rel="first"

Du möchtest dann die URL mit `rel="next"` nehmen und damit den nächsten Request anstoßen.
Dann haben wir auch keine Endlosschleife mit `while True`.

Aber wie gesagt, ist nur eine Idee. Meinetwegen kann es auch so bleiben, wie es ist.

Reply:
Das ist echt mega-viel Arbeit. Mir stört eher, dass es ein fold sein könnte. Am besten soll man einfach eine von beiden nehmen und eigenes Zeug raustun:

https://github.com/sigmavirus24/github3.py
https://github.com/PyGithub/PyGithub

Kann sein, dass die nicht genommen wurden, weil damals irgendwelche Endpunkten wurden noch nicht drin. @stohrendorf  machst du dafür ein Ticket?

Reply:
Ist mir bewusst, dass das der sauberere Weg ist, das ist aber komplizierter umzusetzen.